app/managers.py

```diff
@@ -0,0 +1,31 @@
+import sqlite3
+from models import Actor
+
+
+class ActorManager:
+    def __init__(self):
+        self.conn = sqlite3.connect("cinema.sqlite")
+    
+    def create(self, first_name: str, last_name: str):
+        self.conn.execute("INSERT INTO actors (first_name, last_name)"
+                       "VALUES (?, ?)",
+                       (first_name, last_name))
+        self.conn.commit()
+        self.conn.close()

```

Comment:
Closing the database connection after each operation might not be ideal if you plan to perform multiple operations in sequence. Consider managing the connection lifecycle more effectively, perhaps by opening it in `__init__` and closing it in a separate method or using a context manager.